x-pack/plugins/infra/public/pages/logs/log_entry_rate/sections/anomalies/table.tsx

```diff
@@ -181,7 +178,7 @@ export const AnomaliesTable: React.FunctionComponent<{
         render: (startTime: number) => moment(startTime).format(dateFormat),
       },
       {
-        field: 'datasetName',
+        field: 'dataset',
```

Comment:
Not sure if this is the direct cause, but the table does not render the catch-all dataset as `unknown` anymore:

![image](https://user-images.githubusercontent.com/973741/106891074-81bd9d80-66ea-11eb-824e-4480406c64e3.png)

Reply:
Good catch. This *might* have been me. I found a bug whereby sorting was broken on the dataset column due to the column name being `datasetName`. Will fix regardless 👍